### PR TITLE
migrate to swift4

### PR DESCRIPTION
**Pull request by Igor-Palaguta**

### CODE REVIEW

**Review by 0xTim — CHANGES_REQUESTED** [review]
Can you add a Package@swift-4.swift and alter the CI so that it runs on both 3.1 and 4?

**Comment by Igor-Palaguta** [comment]
Hello @0xTim, please review https://github.com/vapor/validation/pull/12/commits/f5c33c8eac2f9c124860f604ec082e8ad092f82b

File: circle.yml [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
-    - sudo apt-get install vapor
+    - sudo apt-get install swift
     - sudo chmod -R a+rx /usr/
 test:
   override:
     - swift build
     - swift build -c release
     - swift test
+    - sudo apt-get remove swift
+    - sudo apt-get install swift-beta
```
Comment (0xTim) [inline circle.yml]: Could you change this to `sudo apt-get install swift=3.1.1` - it will run on Swift 3 then 🙂

**Review by 0xTim — CHANGES_REQUESTED** [review]

File: circle.yml (hunk 1 above)
Comment (Igor-Palaguta) [reply circle.yml]: @0xTim what version is installed now without version specification `sudo apt-get install swift` 4 or 3?

Comment (0xTim) [reply circle.yml]: Swift 4 now

**Review by 0xTim — APPROVED** [review]

**Comment by tanner0101** [comment]
thanks :)

**Pull request merged**